### ISSUE TITLE
TitleManager 1.6.0 broken commands.

### ISSUE THREAD

**Issue by mibby** [Puharesource/TitleManager#81]
Commands broke as of this commit. https://github.com/Puharesource/TitleManager/commit/0960c4972a37b4ddbbaa135da8e5827d0b87cad3

```
[00:32:23] [Server thread/INFO]: mibby issued server command: /tm bc &btest
[00:32:23] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'tm' in plugin TitleManager v1.6.0
    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[patched_1.10.jar:git-Paper-780]
    at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:152) ~[patched_1.10.jar:git-Paper-780]
    at org.bukkit.craftbukkit.v1_10_R1.CraftServer.dispatchCommand(CraftServer.java:665) ~[patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.PlayerConnection.handleCommand(PlayerConnection.java:1386) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.PlayerConnection.a(PlayerConnection.java:1191) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.PacketPlayInChat.a(PacketPlayInChat.java:45) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.PacketPlayInChat.a(PacketPlayInChat.java:5) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.PlayerConnectionUtils$1.run(SourceFile:13) [patched_1.10.jar:git-Paper-780]
    at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_92]
    at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_92]
    at net.minecraft.server.v1_10_R1.SystemUtils.a(SourceFile:45) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:793) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:403) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:730) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:629) [patched_1.10.jar:git-Paper-780]
    at java.lang.Thread.run(Unknown Source) [?:1.8.0_92]
Caused by: java.lang.NullPointerException
    at io.puharesource.mc.titlemanager.commands.CommandParameters.getServer(CommandParameters.java:56) ~[?:?]
    at io.puharesource.mc.titlemanager.commands.sub.SubBroadcast.onCommand(SubBroadcast.java:59) ~[?:?]
    at io.puharesource.mc.titlemanager.commands.TMCommand.onCommand(TMCommand.java:61) ~[?:?]
    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[patched_1.10.jar:git-Paper-780]
    ... 15 more
[00:32:28] [Server thread/INFO]: mibby issued server command: /tm abc &btest
[00:32:28] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'tm' in plugin TitleManager v1.6.0
    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[patched_1.10.jar:git-Paper-780]
    at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:152) ~[patched_1.10.jar:git-Paper-780]
    at org.bukkit.craftbukkit.v1_10_R1.CraftServer.dispatchCommand(CraftServer.java:665) ~[patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.PlayerConnection.handleCommand(PlayerConnection.java:1386) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.PlayerConnection.a(PlayerConnection.java:1191) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.PacketPlayInChat.a(PacketPlayInChat.java:45) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.PacketPlayInChat.a(PacketPlayInChat.java:5) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.PlayerConnectionUtils$1.run(SourceFile:13) [patched_1.10.jar:git-Paper-780]
    at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_92]
    at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_92]
    at net.minecraft.server.v1_10_R1.SystemUtils.a(SourceFile:45) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:793) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:403) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:730) [patched_1.10.jar:git-Paper-780]
    at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:629) [patched_1.10.jar:git-Paper-780]
    at java.lang.Thread.run(Unknown Source) [?:1.8.0_92]
Caused by: java.lang.NullPointerException
    at io.puharesource.mc.titlemanager.commands.CommandParameters.getServer(CommandParameters.java:56) ~[?:?]
    at io.puharesource.mc.titlemanager.commands.sub.SubABroadcast.onCommand(SubABroadcast.java:51) ~[?:?]
    at io.puharesource.mc.titlemanager.commands.TMCommand.onCommand(TMCommand.java:61) ~[?:?]
    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[patched_1.10.jar:git-Paper-780]
    ... 15 more
```

**Comment by Puharesource** [comment]
I'm fully aware, I simply wanted to commit something that compiled so that I focus on 1.5.12, as I'm currently going through exams as well as having a lot of work to do.

**Comment by mibby** [comment]
All's good. Just thought I'd compile, test, and let you know anyways. :P

**Comment by Puharesource** [comment]
Fixed by commit 03bcec42a15824a55a4306d2e9920285fc73fb32
